src/coreclr/src/libraries-native/CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(GLOBALIZATION_NATIVE_DIR ${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/System.Globalization.Native)
 
 if(CLR_CMAKE_HOST_UNIX AND NOT CLR_CROSS_COMPONENTS_BUILD)
-    add_subdirectory(${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/System.Globalization.Native System.Globalization.Native)
+    include_directories("${CLR_REPO_ROOT_DIR}/src/libraries/Native/Unix/Common")
+    include_directories("${GLOBALIZATION_NATIVE_DIR}")
+
+    add_definitions(-DFEATURE_GLOBALIZATION)
```

Comment:
`FEATURE_GLOBALIZATION` makes it sound like that this is used to enable/disable globalization support in the runtime. It is not the case.

I guess you have added this ifdef to turn the ICU globalization shim on/off. I do not think you need this ifdef. We are working towards having ICU everywhere, and so this ifdef would be eventually enabled everywhere.

For now, I think you can just use `TARGET_UNIX` for the few places - until you do the work to get ICU compiled on Windows too.

Reply:
Yeah. This was a leftover of the initial design I had for the entrypoints.c where I was compiling it as part of System.Globalization.Native instead of libraries-native. Since entrypoints.c is potentially going to be shared across different shims, I added this for that, but it is not needed anymore. Thanks.